srtcore/sync.h

```diff
@@ -534,7 +528,7 @@ class CSync
     // when you don't care whether the associated mutex is locked or not (you
     // accept the case that a mutex isn't locked and the signal gets effectively
     // missed), or you somehow know that the mutex is locked, but you don't have
-    // access to the associated CGuard object. This function, although it does
+    // access to the associated ScopedLock object. This function, although it does
```

Comment:
`UniqueLock`. You can't use `ScopedLock` in automated version of `CSync`.

You might improve this description by mentioning "`UniqueLock` or `Mutex` object".

Reply:
Fixed.